From f1e385a2da37e3b007229ab665b90ebf069448ba Mon Sep 17 00:00:00 2001
From: Artur Wieczorek <artwik@wp.pl>
Date: Fri, 3 Apr 2015 21:48:09 +0200
Subject: [PATCH 3/4] Handle wxDateTime data in wxHandleFatalExceptions (wxMSW)
 only if wxUSE_DATETIME is enabled.

Code using wxDateTime datatype in wxHandleFatalExceptions() should be active only if wxUSE_DATETIME is enabled.
---
 src/msw/main.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/msw/main.cpp b/src/msw/main.cpp
index ad0616b..8f0c91a 100644
--- a/src/msw/main.cpp
+++ b/src/msw/main.cpp
@@ -165,7 +165,11 @@ bool wxHandleFatalExceptions(bool doit)
                             wxT("%s_%s_%lu.dmp"),
                             wxTheApp ? (const wxChar*)wxTheApp->GetAppDisplayName().c_str()
                                      : wxT("wxwindows"),
+#if wxUSE_DATETIME
                             wxDateTime::Now().Format(wxT("%Y%m%dT%H%M%S")).c_str(),
+#else
+                            wxT("?"),
+#endif
                             ::GetCurrentProcessId()
                         );
 
-- 
1.9.5.msysgit.0

